src/deadline/client/ui/__init__.py

```diff
@@ -69,8 +70,9 @@ def gui_context_for_cli():
 
     import click
 
-    has_pyside = importlib.util.find_spec("PySide6") or importlib.util.find_spec("PySide2")
-    if not has_pyside:
+    has_pyside6 = importlib.util.find_spec("PySide6")
```

Comment:
Is this logic change just for code quality purpose or does this actually fix something?

Reply:
Makes it easier to check which pyside to prioritize loading on in the code.